molecule/default/converge.yml

```diff
@@ -32,6 +32,14 @@
     - name: Collect facts
       ansible.builtin.setup:
 
+    - name: Make containerd defautl to crictl
```

Comment:
Do not include the same code block many times. Put that code into top level of molecule folder and use `include_tasks` instead